### ISSUE TITLE
Grantham score : update to latest SnpEff (ANN) annotations

### ISSUE THREAD

**Issue by gpcr** [ashutoshkpandey/Annotation#1]
Hi @ashutoshkpandey Ashutosh, 

It will be great if you can update this to accept latest ANN format of Snpeff

thanks

**Comment by DaRinker** [comment]
Yes. Or why not just untether it from SnpEFF entirely? Modify the script to accept a user generated vcf file of pre-screened missense variants.

**Comment by 15787833237** [comment]
> Yes. Or why not just untether it from SnpEFF entirely? Modify the script to accept a user generated vcf file of pre-screened missense variants.

Could you give me a hand on that? It is too difficult for me ....

**Comment by wpf95** [comment]
Hi
i just use SnpEff 5.1 to annotate vcf file , than i use command (python Grantham_score_calculator.py vcf.file result.txt) to run , but i can not get result .  python give the recall is name"temp2" is not defined .
So, i don not konw how run the Grantham_score_calculator.py to calculator the vcf file.
Can you tell me ? Thank you !
